installer.sh

```diff
@@ -1,10 +1,57 @@
 #!/usr/bin/env bash
+
+systemctl status ModemManager | grep 'active (running)' > /dev/null 2>&1
+
+if [ $? != 0 ]
+then
+        sudo systemctl disable ModemManager > /dev/null
+        sudo systemctl stop ModemManager > /dev/null
+fi
+
 set -e
 
 function info { echo -e "[Info] $*"; }
 function error { echo -e "[Error] $*"; exit 1; }
 function warn  { echo -e "[Warning] $*"; }
 
+# Install required dependencies
+test $? -eq 0 || exit 1 "you should have sudo priveledge to run this script"
+
+info ""
+info ""
+info "Installing Home Assistant dependencies"
+info ""
+info ""
+
+while read -r p ; do sudo apt-get install -y $p ; done < <(cat << "EOF"
+	software-properties-common
+	apparmor-utils 
+	apt-transport-https 
+	avahi-daemon 
+	ca-certificates 
+	curl 
+	dbus 
+	jq 
+	network-manager
+EOF
+)
+
+# Install Docker-ce
+info ""
+info ""
+info "Installing Docker-ce"
+info ""
+info ""
+
+curl -fsSL get.docker.com | sh
```

Comment:
What is it already is installed?

Reply:
I'm not sure the command to check if it is and then skip this step.

This is all just a first attempt cobbled together from research I've been doing.

Reply:
Would this work to check if Docker is installed?

```
    if command -v docker > /dev/null 2>&1; then
        echo "Docker-ce is installed "
      else
        echo "Docker-ce is not present - installing now"
        curl -fsSL get.docker.com | sh
    fi
```